CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ else()
   add_subdirectory(OrbitCaptureGgpService)
   add_subdirectory(OrbitLinuxTracing)
   add_subdirectory(OrbitProducer)
+  add_subdirectory(OrbitProducerSideChannel)
```

Comment:
This thing is Linux-only, but would compile on Windows, right? Should it still be only included on Linux?

Reply:
I don't have an opinion. I made it Linux only because it's conceptually Linux only, as it's about Unix domain sockets. Should it also be build on Windows?

Reply:
oh sorry, didn't noticed it is already in the linux only block. Perfect!